core/lib/resources/presence/index.js

```diff
@@ -170,10 +178,11 @@ Presence.prototype.broadcast = function(message, except) {
   var self = this;
   Object.keys(this.subscribers).forEach(function(subscriber) {
     var client = self.parent.server.clients[subscriber];
-    if(client && client.id != except) {
+    if(client && client.id != except && self.subscribers[client.id] === true) {
```

Comment:
Haha. I just wanted it not to be "true". This was a late night project, will fix this to be clearer. Also, this is strangely in prep for #121